species_files/zebrafish.ini

```diff
@@ -16,7 +16,7 @@ SPECIES_DOWNLOAD_FOLDER: download_files/Zebrafish
 DOWNLOAD: TRUE
 GO_OBO_URL: ftp://ftp.geneontology.org/go/ontology/obo_format_1_2/gene_ontology.1_2.obo
 ASSOC_FILE_URLS: ftp://ftp.geneontology.org/go/gene-associations/gene_association.zfin.gz,
-    http://geneontology.org/gene-associations/gene_association.goa_uniprot_noiea.gz
+    ftp://ftp.geneontology.org/go/gene-associations/goa_uniprot_all_noiea.gaf.gz
```

Comment:
All nine *.ini files are sharing the same 2nd URL. Is it possible to set this URL in a common place (such as `main_config.ini`)? Then next time when you update this URL, you don't have to change all nine files.

Reply:
This change is actually not really trivial since we will want the code in download_files.py and in process_go.py to grab this url from the main_config.ini file instead, download it and process it. It is definitely doable, but it will extend this PR. Do you think I should change it now or create an issue for it?

Reply:
Considering the deadline, let's create an issue for now. Thanks.

Reply:
@rzelayafavila: If you want to also correct the typos in `README` file, please go ahead do it. Thanks.

Reply:
Yep, I am about to push the changes for those.